src/ocamlorg_frontend/pages/home.eml

```diff
@@ -165,11 +165,11 @@ with an emphasis on expressiveness and safety."
               BUILD ANYTHING
             </div>
             <h3 class="text-body-600 font-bold">
-              First class editor and tooling
+              First Class Editor and Tooling
             </h3>
             <div class="my-4 text-lg">
               OCaml has several editor tools that make it easier to program in. Usually, VS Code is recommended for
-              beginners, but there is also good support for Vim and Emacs as well. Tools like the toplevel Utop and
+              beginners, but there is also good support for Vim and Emacs. Tools like the toplevel Utop and
               document generator Odoc are also well supported and used by OCaml programmers all over the world. Highly
```

Comment:
Note: This is suggesting to make a change unrelated to the PR.

As it is now, the frontpage uses unquoted capitalized words in the original font face when referring to tools:
Odoc, Utop (and Emacs and Vim). I didn't write it - so I may or may not agree - but at least it is consistent! 😅
The suggested change would introduce an inconsistency by lower-casing and quoting only one such tool name.

I suggest filing a separate issue/PR and discussing it there.

Reply:
Thank you for the suggestion. We're working making the docs consistent in capitalization across the board, including tool names. When it's the proper name of the tool, it's capitalized, but when it's a command or library name, it's wrapped in `monospace`, if that makes sense. 

Is this what you meant?

Reply:
Yes, why lower-case and quote `odoc` but not  `utop`? https://github.com/ocaml-community/utop
I realize this is a can of worms though (another reason to address it in a separate PR/issue)

Reply:
The `odoc` creator decided that the formal name of the tool is formatted like `odoc` instead of ODoc, which I suggested since it echos OCaml, so it is a rather can of worms.

Reply:
Oh 😬 I'd prefer you address that can in a separate issue/PR... 🙏‍

Reply:
> Thanks @jmid!
> 
> Indeed, we can review the content you didn't change in this PR as part of a separate issue or PR, that sounds better 🙂
> 
> Does it look good on your side for the changes made on the PR @christinerose?

Looks good to me!